openedx/features/course_experience/__init__.py

```diff
@@ -1,8 +1,34 @@
-# Unified course experience settings.
+"""
+Unified course experience settings and helper methods.
+"""
+
+import waffle
+
+from request_cache.middleware import RequestCache
 
 # Waffle flag to enable a single unified "Course" tab.
 UNIFIED_COURSE_EXPERIENCE_FLAG = 'unified_course_experience'
 
 # Waffle flag to enable the full screen course content view
 # along with a unified course home page.
 UNIFIED_COURSE_VIEW_FLAG = 'unified_course_view'
+
+
+def main_course_url_name(request=None):
+    """
+    Returns the main course URL for the current user.
```

Comment:
Thanks. I agree except I want to keep "for the current user". Once we drop the Waffle flag this method can be removed, and until then it is important.